solidity/dashboard/src/components/DelegateStakeForm.jsx

```diff
@@ -123,27 +131,21 @@ const TokensAmountField = ({
           normalize={normalizeAmount}
           format={formatFormAmount}
           placeholder="0"
-          instructionText={`The minimum stake is ${displayAmount(
-            minStake
-          )} KEEP`}
+          additionalInfoText={`MIN STAKE ${displayAmount(minStake)} KEEP`}
           leftIcon={<Icons.KeepOutline className="keep-outline--mint-100" />}
           inputAddon={
             <MaxAmountAddon onClick={onAddonClick} text="Max Stake" />
           }
+          tooltipText={
+            <>
+              The minimum stake will decrease to 50,000 KEEP on 04/28/2021. You
```

Comment:
Can we create a kind of helper that returns the next minimum stake value and date according to the https://staking.keep.network/about-staking/staking-minimums ? Otherwise, we will have to update manually each time.